Bookish.DataAccess/BookishService.cs

```diff
@@ -41,23 +41,26 @@ GROUP BY
             return dbConnection.Query<CatalogueEntry>(query, new { Filter = $"%{filter}%" });
         }
 
-        public IEnumerable<CopiesEntry> GetCopies(string filter)
+        public IEnumerable<BookCopy> GetCopies(string filter)
```

Comment:
One thing I didn't notice earlier, `filter` isn't a very descriptive name in this case. I think `isbn` is more descriptive 🙂